Makefile

```diff
@@ -0,0 +1,50 @@
+SHELL := /bin/bash -o pipefail
```

Comment:
What's this Makefile?

Reply:
Ughh, it's from our internal build system, it shouldn't be here :( 
Anyway, I've rewritten commit and removed config option